packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -219,6 +221,14 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu>, IWith
 
   /** Method to call when trying to render a submenu. */
   onRenderSubMenu?: IRenderFunction<IContextualMenuProps>;
+
+  /**
+   * Method to override the render of the children menu items
+   * @default ContextualMenuItemChildren
+   */
+  contextualMenuChildrenAs?:
```

Comment:
can we change Children to child? contextualMenuChildAs, IContextualMenuItemChildProps etc - It's a single item, not an array.

Reply:
Actually, can we remove children/child completely, and just call it a ContextualMenuItem?

Reply:
I'm happy to change them to whatever. @joschect suggested the current names, @joschect do you have any objections to changing them as Micah suggests?

Reply:
contextualMenuChildItemAs would be fine. Just change the props to reflect that as well.